core/app_switcher/app_switcher.py

```diff
@@ -254,6 +254,13 @@ def get_running_app(name: str) -> ui.App:
                 return application
         raise RuntimeError(f'App not running: "{name}"')
 
+    def switch_last_focused():
```

Comment:
Having an if statement here instead of a new context seems OK to me for now.

I'd just suggest we change the action name to switcher_focus_last_window() to better match with the existing action names.